p/api/greader.php

```diff
@@ -261,8 +261,8 @@ function subscriptionList() {
 
 	$pdo = new MyPDO();
 	$stm = $pdo->prepare('SELECT f.id, f.name, f.url, f.website, c.id as c_id, c.name as c_name FROM `%_feed` f
-		INNER JOIN `%_category` c ON c.id = f.category');
-	$stm->execute();
+		INNER JOIN `%_category` c ON c.id = f.category AND f.priority >= :priority_normal');
```

Comment:
Ups. Typo here. It is a `WHERE`, not an `AND`.

Reply:
It's on purpose. The inner join is done on the category and the priority